lib/main.js

```diff
@@ -128,8 +136,7 @@ module.exports = class Client {
         assert(
             files.every(file => file.includes('.js')) ||
                 files.every(file => file.includes('.css')),
-            `Expected ALL items in array 'files' to end with .js 
-                or ALL items in array 'files' to end with .css, got ${files}`
+            `Expected ALL items in array 'files' to end with .js or ALL items in array 'files' to end with .css, got ${files}`
```

Comment:
can we do `files.join(', ')`? instead of relying on a `toString`?

Reply:
done